packages/mobile-top-nav/README.md

```diff
@@ -0,0 +1,77 @@
+# \<topnav-element>
+
+The mobile navigation menu for archive.org
+
+## Installation
+
+```bash
+yarn add @internetarchive/mobile-top-nav
+```
+
+## Usage
+
+```js
+// ia-mobile-top-nav.js
+import IAMobileTopNav from '@internetarchive/mobile-top-nav';
+export default IAMobileTopNav;
+```
+
+```html
+<!-- index.html -->
+<script type="module">
+  import './ia-mobile-top-nav.js';
+</script>
+
+<style>
+  /* Defaults */
+  topnav-element {
+    --white: #fff;
```

Comment:
Is this demo code correct?

Reply:
What do you mean by correct? The colors match the design. As for variable naming, I haven't made changes to what Kanchan originally developed. While the initial variable names don't tell you what they're styling, setting variables for use cases based on these colors makes sense. `--primary-text-color` is one example. That way, the base theme colors are set and individual uses can be detached from a given color and reassigned as design changes.

If, instead, we create 8 variables and each are set to `#fff`, when we want to change that color across the entire theme, we have to update 8 references. The same repetition is already present in Petabox's color variables (`@color-white` and `@white`, `@body-text-color` and `@grey20`, etc.)

Reply:
By correct, I mean are these the CSS variables that we're passing through or is this code leftover from Kanchan?

I don't think CSS variables should be named literally for the color that they're representing. It is confusing to take a name like `white` and redefined it to something other than `#fff`. Similar with `grey999`.. it doesn't make any sense to be anything other than `#999`.

CSS variables should be semantically meaningful and specific like `navigation-background-color`. I shouldn't have to know the internals of your component to have an idea of how the colors are being used.

Regarding updating 8 different places that use "`white`".. I don't think we should ever blanket update a color. We should treat each use on a case-by-case basis since it means different things in different comments.